docs/README.md

```diff
@@ -1,57 +1,36 @@
-# MLflow Docusaurus Migration
+# MLflow Documentation
 
-The `docusaurus` branch is the working branch for all Docusaurus migration work. We are switching away from Sphinx for our main informational docs, and using [Docusaurus](https://docusaurus.io/) instead.
+This README covers information about the main MLflow documentation. The API reference is built separately and included as a static folder during the full build process. Please check out the [README](https://github.com/mlflow/mlflow/blob/master/docs/api_reference/README.md) in the `api_reference` folder for more information.
 
-## Jobs to be Done
+## Prerequisites
 
-1. Migrating all non-API related docs from `.rst` to `.mdx`, the Docusaurus format
-  a. These files can be found in the `rst_to_migrate` folder
-2. Migrating notebooks to be rendered via [this plugin](https://github.com/datalayer/jupyter-ui/tree/main/packages/docusaurus-plugin), rather than by `nbsphinx`
-  a. Notebooks are also found in the `rst_to_migrate` folder, but will end in `.ipynb` 
+**Necessary**
+- NodeJS >= 18.0 (see the [NodeJS documentation](https://nodejs.org/en/download) for installation instructions)
 
-### Installation
+**Optional**
+- (For building MDX files from `.ipynb` files) Python 3.9+ and [nbconvert](https://pypi.org/project/nbconvert/).
+- (For building API docs) See [doc-requirements.txt](https://github.com/mlflow/mlflow/blob/master/requirements/doc-requirements.txt) for API doc requirements.
 
-```
-$ yarn
-```
-
-### Local Development
-
-```
-$ yarn start
-```
-
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
-
-### Build
+## Installation
 
 ```
-$ yarn build
+yarn
 ```
 
-This command generates static content into the `build` directory and can be served using any static contents hosting service.
-
-### Formatting
+## Local Development
 
 ```
-$ yarn format
+yarn start
 ```
 
-This command automatically formats all `.mdx` files within the `/docs` folder
-
-
-### Deployment
-
-Using SSH:
-
-```
-$ USE_SSH=true yarn deploy
-```
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
```

Comment:
Let's add a note here saying some features might be disabled in this mode.

Example: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects